### PR TITLE
Avatar store data

### PR DESCRIPTION
**Pull request by thoys**
# QA Test

NOTE: use the domain hifi://dev-avatarisland (the A&J store) and the latest https://www.highfidelity.com/dev-download to test this PR

Test Rail available here:

https://highfidelity.testrail.net/index.php?/cases/view/1580&group_by=cases:section_id&group_order=asc&group_id=1457

### CODE REVIEW

File: DomainContent/AvatarStore/request.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+// TODO: should we make an exception for the modules folder? (Thoys Jan-10-2018)
+
+
+// ===========================================================================================
+module.exports = {
+
+    // ------------------------------------------------------------------
+    request: function (options, callback) { // cb(error, responseOfCorrectContentType) of url. A subset of npm request.
+            var httpRequest = new XMLHttpRequest(), key;
```
Comment (misslivirose) [inline DomainContent/AvatarStore/request.js]: This seems indented one too far?

**Comment by QAadahlin** [comment]
Currently testing

**Comment by QAadahlin** [comment]
Testing completed. FAILED. Miniatures of items did not show up at checkout. Also, they did not appear in the checkout screen on the tablet. The tablet only displayed instructions. See image below:

**Comment by QAadahlin** [comment]
@conklin94122 Testing completed for PR74. FAILED. Miniatures of items did not show up at checkout. Also, they did not appear in the checkout screen on the tablet. The tablet only displayed instructions. See image below:
checkout

**Comment by QAadahlin** [comment]
@conklin94122 Re-tested #74. PASSED.

**Pull request merged**